### PR TITLE
TELCODOCS-905: expanding networkType explanation

### PR DESCRIPTION
**Pull request by tmulquee**
Version(s): 4.11, 4.10, 4.9.

Issue:  https://issues.redhat.com/browse/TELCODOCS-905

Link to docs preview: https://53956--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_bare_metal/installing-bare-metal-network-customizations.html#installation-configuration-parameters-network_installing-bare-metal-network-customizations

Search for "OVNKubernetes".

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/installation-configuration-parameters.adoc [hunk 1]
```diff
@@ -359,7 +359,7 @@ ifdef::openshift-origin[]
 Either `OpenShiftSDN` or `OVNKubernetes`. The default value is `OVNKubernetes`.
 endif::openshift-origin[]
 ifndef::openshift-origin[]
-Either `OpenShiftSDN` or `OVNKubernetes`. The default value is `OpenShiftSDN`.
+Either `OpenShiftSDN` or `OVNKubernetes`. `OpenShiftSDN` is the provider for all-Linux networks. `OVNKubernetes` is the provider for hybrid networks that contain both Linux and Windows servers. The default value is `OpenShiftSDN`.
```
Comment (asood-rh) [inline modules/installation-configuration-parameters.adoc:362]: In 4.12 default plugin is OVNKubernetes.

Comment (anuragthehatter) [reply modules/installation-configuration-parameters.adoc:362]: This should be ok as this change goes in 4.11 and lower. How about `OpenShiftSDN is the provider for all-Linux networks. OVNKubernetes is the provider for linux networks as well as for hybrid networks that contain both Linux and Windows servers` ?

**Comment by tmulquee** [comment]
Thanks Anurag, I like this wording.

**Comment by tmulquee** [comment]
I've gone with "Either `OpenShiftSDN` or `OVNKubernetes`. `OpenShiftSDN` is a CNI provider for all-Linux networks. `OVNKubernetes` is a CNI provider for Linux networks and hybrid networks that contain both Linux and Windows servers. The default value is `OpenShiftSDN`."
As Anurag mentioned, this is only for 4.11 and prev.

**Comment by tmulquee** [comment]
@anuragthehatter @asood-rh the updated text is available for your attention in the doc preview above. 
Please apply /lgtm labels if you are satisfied.

File: modules/installation-configuration-parameters.adoc [hunk 2]
```diff
@@ -359,7 +359,7 @@ ifdef::openshift-origin[]
 Either `OpenShiftSDN` or `OVNKubernetes`. The default value is `OVNKubernetes`.
 endif::openshift-origin[]
 ifndef::openshift-origin[]
-Either `OpenShiftSDN` or `OVNKubernetes`. The default value is `OpenShiftSDN`.
+Either `OpenShiftSDN` or `OVNKubernetes`. `OpenShiftSDN` is a CNI provider for all-Linux networks. `OVNKubernetes` is a CNI provider for Linux networks and hybrid networks that contain both Linux and Windows servers. The default value is `OpenShiftSDN`.
```
Comment (asood-rh) [inline modules/installation-configuration-parameters.adoc:362]: @tmulquee If this documentation is going for 4.12 release then default value is OVNKubernetes. Rest looks good.

**Review by jeana-redhat — COMMENTED** [review]
Content and format LGTM. 

Since this is only going into 4.9-4.11, it should not go into `main` (I am assuming there is another PR for 4.12+ that was/will be merged into `main` and cherrypicked to 4.12?). So, should open new PRs for each branch, but I will leave the peer review done so you can reference this and go straight to merge review with those.

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

**Comment by tmulquee** [comment]
Yes. In 4.12 the default changes to OVN Kubernetes, which is why this PR
doesn't reference 4.12.

**Comment by EricPonvelle** [comment]
@tmulquee You will need to do manual cherrypicks for enterprise-4.9 - 4.11

**Comment by jboxman-rh** [comment]
@jeana-redhat @EricPonvelle @tmulquee 

Looks like this nonetheless did got into `main`. This might need to be fixed?

**Comment by tmulquee** [comment]
Yes. The entry in main needs to be edited to reflect that in 4.12 the
default is now OVN Kubernetes,

**Comment by bergerhoffer** [comment]
@tmulquee It looks like this PR didn't go into 4.12. If a PR is targeting main, then it needs to also go into 4.12.

I kicked off the cherry pick to 4.12 and labeled this PR for 4.12 as well. I'm going to merge it.

Can you please look into whether the default here needs to be changed to `OVNKubernetes` for main/4.12 in a separate PR? Or @jboxman-rh, do you already have a PR that is taking care of that update?

**Comment by jboxman-rh** [comment]
@bergerhoffer, so, I don't know much about this, but it conflicted with my last cherry-pick, so I tracked it down.

> Either `OpenShiftSDN` or `OVNKubernetes`. `OpenShiftSDN` is the provider for all-Linux networks. `OVNKubernetes` is the provider for hybrid networks that contain both Linux and Windows servers. The default value is `OpenShiftSDN`.

The difference between this and `main`/`enterprise-4.12` is likely that the default value is different. Probably needs a different PR to fix it for `main` and `enterprise-4.12`. And the text was entirely missing in 4.12, because it hit `main` but not 4.12.

Or something like that.

**Comment by tmulquee** [comment]
This PR was never intended to hit main or 4.12. See the Version(s) setting in the summary.
As we are all aware, the default setting for this parameter changes in 4.12, and there is a separate epic for this work.

**Comment by bergerhoffer** [comment]
@tmulquee If this wasn't meant for main/4.12, then the PR should not have been submitted against the main branch. It should have gone straight into the enterprise version branch(es) that it applied to.

If this update truly does not belong in main/enterprise-4.12, please hit the "revert" button and submit that new PR for main/enterprise-4.12 to pull them out. And then continue your epic work as needed. Or if this change is fine as it is, then submit the follow-up work for your epic as needed. Let me know if you have any questions on it, thanks!

**Comment by tmulquee** [comment]
I've created a new PR to fix the content that was unintentionally merged to main during the merge review process. The only change required is to state that the default is OVNKubernetes in main. We can then cherrypick from that version of the text to 4.12.

I will reach out to Jason to ensure he is aware of this fix.

https://github.com/openshift/openshift-docs/pull/54718

**Pull request merged**